docs/build-environment.yml

```
@@ -14,8 +14,8 @@ dependencies:
   - myst-parser
   - pydata-sphinx-theme
   - jupyterlab >=4.0.5,<5
-  - jupyterlite-core >=0.2.0a1,<0.3.0
-  - jupyterlite-sphinx >=0.9.1
+  - jupyterlite-core >=0.3.0,<0.4.0
```

Does it need to be >=0.3?

```suggestion
  - jupyterlite-core >=0.2,<0.4
```

---

For the docs I guess we can use the latest stable?

---

> For the docs I guess we can use the latest stable?

Although we would need to wait for the new version to land on conda-forge.

---

Changed to `>=0.2,<0.4` for consistency with what is specified in `pyproject.toml`